### ISSUE TITLE
installation/Upgrade  could not be started from 1.2.19 to 1.2.20

### ISSUE THREAD

**Issue by gj00354347** [Cacti/cacti#4734]
# Describe the bug

HI Team ,

after putting the new cacti directory in right location and then transferring all the requisite files and folders like old-cacti/scripts/* cacti-1.2.20/scripts/ , old-cacti/resource/* cacti-1.2.20/resource/ , old-cacti/plugins/* cacti-1.2.20/plugins/ and then copying the old-cacti/include/config.php cacti-1.2.20/include/config.php .

i tried to open new cacti so that it takes me to installation page , the authentication which is done via LDAP is OK but after that in url i see it is trying to take me to install/install.php but then error 500 appears .

yes wherever i have mentioned cacti-1.2.20 that is cacti and when i revert to cacti 1.2.19 all is ok . i mean mv cacti cacti-1.2.20 and mv cacti-1.2.19 cacti and then try to access and all is ok .

yes i have cleared all browser history and cookie still i am facing this problem .
PFB snapshot 

![image](https://user-images.githubusercontent.com/27295357/164242171-ba3a45f0-db27-4fda-b6cf-5ea962ab1188.png)

**Comment by gj00354347** [comment]
Hi Vinten ,

just to add this is what I see in ssl_error_log ,

[Thu Apr 28 11:44:40.087491 2022] [:error] [pid 87273] [client IP:64855] PHP Parse error:  syntax error, unexpected '[' in /opt/SP/cacti/cacti/lib/api_device.php on line 1214, referer: https://main-poller/cacti/install/install.php

**Comment by netniV** [comment]
Hi @gj00354347, this has already been found and fixed.  Not sure if I wasn't clear about that.  It affects the installer because it utilise the api.

**Comment by gj00354347** [comment]
thanks Vinten , any way how can I circumvent or bypass this because it's getting stuck there . 

and is there any requirement like PHP version should be 7 or something like that ??

**Comment by netniV** [comment]
No, Though we may end up getting there given how long 5.4 has been out of support now.  If you search through the recent issues, I believe you'd have found it reported in issues #4730, #4728, #4724 but actually fixed in #4688
